app/models/effective/form_input.rb

```diff
@@ -180,13 +178,10 @@ def build_feedback
 
       invalid = object.errors[name].to_sentence.presence if object.respond_to?(:errors)
       invalid ||= options[:feedback][:invalid].delete(:text).presence
-      invalid ||= [("can't be blank" if options[:input][:required]), ('must be valid' if validated?(name))].tap(&:compact!).join(' and ').presence
-      invalid ||= "can't be blank or is invalid"
+      invalid ||= [("Can't be blank" if options[:input][:required]), ('must be valid' if validated?(name))].tap(&:compact!).join(' and ').presence
+      invalid ||= "Can't be blank or is invalid"
```

Comment:
FYI, these are the first render feedback messages.  If a resource is saved and invalid, renders the #edit view, these placeholder feedback will not be presented, and instead the server side validations will be.  It will say the rails default "can't be blank" with the lower case C.

So these changes mean on first form view, the "Can't" will be used, then if the form is invalid, "can't" might still be used

Reply:
i guess i should also add `text-transform: capitalize;` to the feedback css then, just pushed a change for that